ui/app/AppLayouts/Wallet/controls/ManageTokensGroupDelegate.qml

```diff
@@ -71,31 +90,95 @@ DropArea {
+                visible: root.unknown
+                asset.name: "help"
+                asset.color: Theme.palette.baseColor1
+                asset.isImage: false
+            }
+
             StatusRoundedImage {
+                visible: !root.unknown
                 radius: root.isCollection ? Style.current.radius : height/2
-                Layout.preferredWidth: root.isCollectible ? 44 : 32
-                Layout.preferredHeight: root.isCollectible ? 44 : 32
-                image.source: root.isCollection ? model.imageUrl : model.communityImage // FIXME unify group image
+                Layout.preferredWidth: {
+                    if (!visible){
+                        return 0
+                    }
+                    return root.isCollectible ? 44 : 32
+                }
+                Layout.preferredHeight: {
+                    if (!visible){
+                        return 0
+                    }
+                    return root.isCollectible ? 44 : 32
+                }
+                image.source: {
+                    if(root.isCollection) {
+                        return model.imageUrl
+                    }
+                       
+                    return model.communityImage // FIXME unify group image
+                }
                 showLoadingIndicator: true
                 image.fillMode: Image.PreserveAspectCrop
             }
 
-            StatusBaseText {
+            RowLayout {
+                id: communityNameRow
+                spacing: 2
                 Layout.fillWidth: true
-                text: groupedCommunityTokenDelegate.title
-                elide: Text.ElideRight
-                maximumLineCount: 1
-                font.weight: Font.Medium
+
+                StatusBaseText {
+                    text: groupedCommunityTokenDelegate.title
+                    elide: Text.ElideRight
+                    maximumLineCount: 1
+                    font.weight: Font.Medium
+                }
+
+                CopyToClipBoardButton {
+                    visible: root.unknown && tooltipMouseArea.containsMouse
+                    icon.height: Theme.primaryTextFontSize
+                    icon.width: Theme.primaryTextFontSize
+                    icon.color: Theme.palette.directColor1
+                    Layout.preferredWidth: 16
+                    Layout.preferredHeight: 16
+                    color: Style.current.transparent
+                    textToCopy: model.communityName
+                    onCopyClicked: {
+                        root.walletStore.copyToClipboard(textToCopy)
+                    }
+                }
+
+                StatusToolTip {
+                    id: communityNameToolTip
+                    text: qsTr("Community name could not be fetched")
+                    visible: root.unknown && tooltipMouseArea.containsMouse
+                    orientation: StatusToolTip.Orientation.Top
+                }
+            }
+
+            MouseArea {
```

Comment:
You don't need this `MouseArea`; the `groupedCommunityTokenDelegate` (aka `StatusDraggableListItem`) is a `Control` and therefore has a `hovered` property

Reply:
I only want the hovered on this specific row layout. not the whole draggbale area
Any other way?

Reply:
Ok, then just a `HoverHandler` is enough; nesting `MouseAreas` tend to conflict, esp. when using drag'n'drop